packages/common/src/lib/action/actionbar/actionbar.component.ts

```diff
@@ -143,11 +162,43 @@ export class ActionbarComponent implements OnDestroy, OnChanges {
     return this.horizontal;
   }
 
+  get heightCondition(): boolean {
+    if (this.withTitle === false) {
```

Comment:
I get why withTitle needs to be false in the context of our sidenav but I think it would be better to add an input to let the parent component control that behavior. What do you think?

Reply:
Another input could effectively do the job, but since withTitle do it perfectly, wouldn't be redundant to put another one to do exactly the same job ?

Reply:
Giving too many responsabilities (that are not clearly related) to an input makes it difficult to reason about it. Also, if this feature was extended to support horizontal actionbar, it would be easy to justify having those buttons as well as a title.